OpenSim/Analyses/JointReaction.h

```diff
@@ -72,12 +72,12 @@ OpenSim_DECLARE_CONCRETE_OBJECT(JointReaction, Analysis);
         /* Joint reference*/
         const Joint* joint;
         /* The body upon which the force is applied */
-        const PhysicalFrame* appliedOnBody;
+        const Frame* appliedOnBody;
         /* Is the reaction force applied on the Body the child or the parent
            of the joint?*/
         bool isAppliedOnChild;
         /* The reference Frame in which the force should be expressed */
-        const PhysicalFrame* expressedInFrame;
+        const Frame* expressedInFrame;
```

Comment:
This was again due to `findBaseFrame()` returning a `Frame` rather than having all of the words right. Because of this, the comment may also not want to reference a "Frame"  since JointReaction analysis couldn't really do any arbitrary Frame either (and is now specifically the "base frame" of either "parent" or "child"). Let me know what you think is best.